sphinx/docserver.py

```diff
@@ -35,7 +41,7 @@ def send_alert():
 @app.route('/en/latest/<path:filename>')
 def send_docs(filename):
     return flask.send_from_directory(
-        os.path.join(_basedir, "sphinx/build/html/"), filename)
+        os.path.join(_basedir, "sphinx/_build/html/"), filename)
```

Comment:
@nimishbongale why this change? The subdir is named `build` on my local machine

Reply:
@bryevdv it worked with `_build` in my machine. The folder that `sphinx-build` generated was named `_build`, and hence the change. Let me know if there's something wrong!

Reply:
Yeah I checked and the Makefle specifies `BUILDDIR    = build` so I think this change needs to be reverted

Reply:
@bryevdv fixed it.